dash/frontend/src/app/core/route-guards/auth.guard.ts

```diff
@@ -12,7 +12,7 @@ import {environment} from '../../../environments/environment';
 @Injectable({
   providedIn: 'root'
 })
-export class AuthGuard implements CanActivate {
```

Comment:
Since this is deprecated (taking it out is the right call of course) -
Did we double-check that we're implementing it as a functional guard?
[link to the documentation with relevant portion highlighted](https://angular.io/api/router/CanActivate#:~:text=Deprecated%3A%20Class%2Dbased%20Route%20guards%20are%20deprecated%20in%20favor%20of%20functional%20guards.%20An%20injectable%20class%20can%20be%20used%20as%20a%20functional%20guard%20using%20the%20inject%20function%3A%20canActivate%3A%20%5B()%20%3D%3E%20inject(myGuard).canActivate()%5D.)

Reply:
@becky-intelletive you are right. We used CanActivateFn instead of CanActivated. Please review and let me know if changes are required.

Reply:
@gtarique - Have you tested it before and after the changes?
ex -

1. a logged-out user shouldn't have access to cluster pages
2. a non-super-admin shouldn't have access to manage API Keys

etc :)

Reply:
@becky-intelletive I tested and it's working fine.